test/test_tensor.h

```diff
@@ -927,11 +927,11 @@ TEST(tensor, nd1) {
 }
 
 TEST(tensor, print) {
-    Tensor<float_t, 4> t({3,2,2,2});
+    Tensor<float_t, 5> t({3,2,2,2,2});
```

Comment:
We'd be able to check the printing routine further by testing many variations of tensors.
But well... I wonder if there is any specific plans or goals for the printing routine.